tests/eas/load_tracking.py

```diff
@@ -537,6 +540,206 @@ def test_load_avg_behaviour(self, experiment, tasks):
         """
         return self._test_behaviour(experiment, tasks, 'load_avg')
 
+class _CPUMigrationBase(LisaTest):
+    """Base class for shared functionality of tasks migration tests"""
+    test_conf = {
+        'tools'    : [ 'rt-app' ],
+        'ftrace' : {
+            'events' : [
+                'sched_switch',
+                'sched_load_cfs_rq',
+            ],
+        },
+        'modules': ['cpufreq', 'cgroups'],
+    }
+
+    @classmethod
+    def setUpClass(cls, *args, **kwargs):
+        super(_CPUMigrationBase, cls).runExperiments(*args, **kwargs)
+
+    @classmethod
+    def generate_json(cls, test_env, tasks, json_file_name):
+        """
+        Generate a json file which describes the different tasks.
+        :param test_env: Environment of the test
+        :type test_env: TestEnv
+
+        :param tasks: lists of task
+        :type tasks:[RTATask]
+
+        :param json_file_name: name of the json file
+        :type json_file_name: str
+        :return the generate file name without extensin
```

Comment:
That's a bit confusing... looking at rtapp.conf() code it just returns a "test label"... and that's not the name the "generated json" file name..

Reply:
If that happens to be the same name of the generated json file (to be checked) then why not adding the ".json" extension here... or even better... return the full path of the generated json file.
This would match better with the method name... and simplify callsites code.

Reply:
Done: change to return the json file name from rtapp (rtapp.json).